test/fixtures/IL-2022.json

```diff
@@ -0,0 +1,290 @@
+[
+  {
+    "date": "2022-01-17 00:00:00 -0600",
+    "start": "2022-01-16T16:00:00.000Z",
```

Comment:
On January 16, the sunset in Jerusalem is at 16:59.

Reply:
Please check section "Dates in Hebrew calendar" of specification.md